src/Endpoint/EndpointBase.php

```diff
@@ -281,13 +307,23 @@ protected function getRequestMultipartStreamElements(): array
     }
 
     /**
+     * Override this method to define the simple form elements.
+     *
      * @return string[]
      */
     protected function getRequestFormElements(): array
     {
         return [];
     }
 
+    /**
+     * Override this method to define the body of the request as JSON.
```

Comment:
```suggestion
     * Gets the ...
     *
     * Override this method to provide a JSON body for a request.
```
The short description should always start with the "Action" of the function name, Gets, Sets, Adds, etc
Do this for all pls and then the more detailed explanation can be the "Override.." part